index.html

```diff
@@ -0,0 +1,203 @@
+                <span class="form-section__name">
+                    <span class="name">
+                        <label class="form__label label">First Name</label>
+                        <input class="form__input input"
+                            type="text"
+                            placeholder=""
+                            required
+                            minlength="2"
+                            maxlength="30">
+                    </span>
+                    <span class="name">
+                        <label class="form__label label">Last Name</label>
+                        <input class="form__input input"
+                            type="text"
+                            placeholder=""
+                            required
+                            minlength="2"
+                            maxlength="30">
+                    </span>
+                </span>
+                <span class="form-section__email">
+                    <span class="email">
+                        <label class="form__label label">Email</label>
+                        <input class="form__input input"
+                            type="email"
+                            placeholder=""
+                            required
+                            minlength="2"
+                            maxlength="30">
+                    </span>
+                    <span class="email">
+                        <label class="form__label label">Legal Gender</label>
+                        <select class="form__select select" name="gender" id="gender">
+                            <option value="1" selected>Male</option>
+                            <option value="2">Female</option>
+                            <option value="3">Other</option>
+                        </select>
+                    </span>
+                </span>
+            </form>
+        </div>
+    </div>
+    <div class="form__info info info--second">
+        <div class="form__subtitle subtitle">
+            <div class="subtitle__logo logo">
+                <span class="logo__img img-logo">
+                    <img class="img" src="icons/icon-second.svg" alt="icon">
+                </span>
+                <h2 class="logo__title">Work information</h2>
+            </div>
+            <p class="subtitle__par par">Basic information about your role</p>
+        </div>
+        <div class="form__form form-main">
+            <form class="form__form-section form-section form--second">
+                <span class="form-section__job">
+                    <span class="job">
+                        <label class="form__label label label--manager">Your Manager</label>
+                        <input class="form__input input input--manager"
+                            type="text"
+                            placeholder=""
+                            required
+                            minlength="2"
+                            maxlength="30">
```

Comment:
`readonly` or `disabled`
<img width="432" alt="image" src="https://user-images.githubusercontent.com/40409129/204769349-0a2326e8-f692-407c-b0b5-ca7ef0614946.png">

Reply:
you can remove modification class `input--manager` and add `:disabled` or `:read-only` styles

Reply:
🐛Fix it!